src/stylesheets/application.scss

```diff
@@ -0,0 +1,16 @@
+// Variables
+@import 'variables';
+
+// Dependencies
+@import '~bootstrap/scss/bootstrap';
+
+// Functions
+@import 'functions/sizing';
+
+// Base
+
+// Layouts
+
+// Components
```

Comment:
@carryall Not sure if we will include css components in this file as this file(application.scss) included in the global style.scss file and this component css will be considered as global css. And Angular create a component with html, css and js file under a folder and those component css will apply only to that component. What do think to remove the comment and not placing component css file in here?